repo/packages/C/confluent-connect/2/resource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "assets": {
+    "container": {
+      "docker": {
+        "image": "dbtucker/cp-kafka-connect"
```

Comment:
+1 we really need the image to be a single variable (because of downstream tooling).